src/kestrel_datasource_stixshifter/interface.py

```diff
@@ -206,19 +206,24 @@ def query(uri, pattern, session_id, config):
 
                     result_retrieval_offset = 0
                     has_remaining_results = True
+                    pagesize = transmission.get_pagesize()
```

Comment:
Shall we remove these two lines to minimize stix-shifter API change?

Reply:
Removed in bbb1bfebbb17e5e858c6b7791ea1659938fdd203